.chloggen/deprecate_access_token_passthrough.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: deprecation
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: sapmreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Deprecate the use of `splunk.AccessTokenPassthroughConfig` config field in favor of using include_metadata to pass token in request context.
+
+# Mandatory: One or more tracking issues related to the change. You can use the PR number here if no issue exists.
+issues: []
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Deprecating the use of `splunk.AccessTokenPassthroughConfig` config field means that it is recommended to use the include_metadata option to pass the token in the request context.
```

Comment:
Please also add a line that the batch processor should be updated as well, even if we will do it automatically. The same for the warning message